packages/turf-distance/index.ts

```diff
@@ -1,13 +1,61 @@
 import { Point } from "geojson";
 import { getCoord } from "@turf/invariant";
-import { radiansToLength, degreesToRadians, Coord, Units } from "@turf/helpers";
+import {
+  radiansToLength,
+  degreesToRadians,
+  datums,
+  Coord,
+  Datum,
+  Units,
+  convertLength,
+} from "@turf/helpers";
+const LatLon = require("geodesy").LatLonEllipsoidal;
```

Comment:
Can this be switched to an import?

Reply:
I see, you're just using the same logic as was there before.

Reply:
Yep, this has been converted to an import in the latest commit.